tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -5081,6 +5081,13 @@ Status ConvertTopK(OpConverterParams* params) {
       CheckInputsWeights(*params, {{"input", false}, {"k", true}}));
   TF_RETURN_IF_ERROR(
       AllowDataTypes(*params, {DataType::DT_FLOAT, DataType::DT_HALF}));
+  TFAttrs attrs(node_def);
+  const bool sorted = attrs.get<bool>("sorted");
+  if (!sorted) {
+    return errors::InvalidArgument("Only sorted=True is supported, at",
```

Comment:
Do you know what the semantic is when sorted is false? TF API doc doesn't say anything about that, that means it can be in any order? If that's the case, sorted order is also a valid order for sorted=false?

Reply:
My guess is that the `k` returned values are sorted by index in case of `sorted=False`.
Let me see if I can find that quickly in the op definition.

Reply:
Yeah, I think they are sorted by index according to https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/topk_op.cc#L190-L207

Reply:
Thanks, but is that part of the API semantic? I could not find anything in https://www.tensorflow.org/api_docs/python/tf/math/top_k saying that. If that's not an API guarantee, it means any order is fine?

For example, did you see not fixing this causing any problem for any models?

Reply:
You are right, looks like there is no guarantee in the API in case of `sorted=False`.

I didn't find any model failing. I just see that TensorRT only supports `sorted=True`, and I thought we should check for that.